cmd/server/entrypoint.go

```diff
@@ -5,26 +5,37 @@ import (
 	"encoding/json"
 	"fmt"
 
+	"github.com/Layr-Labs/eigensdk-go/logging"
+
+	"github.com/Layr-Labs/eigenda-proxy/common"
 	"github.com/Layr-Labs/eigenda-proxy/flags"
 	"github.com/Layr-Labs/eigenda-proxy/metrics"
 	"github.com/Layr-Labs/eigenda-proxy/server"
-	"github.com/ethereum/go-ethereum/log"
+	eigenda_common "github.com/Layr-Labs/eigenda/common"
 	"github.com/urfave/cli/v2"
 
 	"github.com/ethereum-optimism/optimism/op-service/ctxinterrupt"
-	oplog "github.com/ethereum-optimism/optimism/op-service/log"
 )
 
 func StartProxySvr(cliCtx *cli.Context) error {
-	log := oplog.NewLogger(oplog.AppOut(cliCtx), oplog.ReadCLIConfig(cliCtx)).New("role", "eigenda_proxy")
-	oplog.SetGlobalLogHandler(log.Handler())
+
+	logCfg, err := common.ReadLoggerCLIConfig(cliCtx, "log")
+	if err != nil {
+		return err
+	}
+
+	log, err := eigenda_common.NewLogger(*logCfg)
```

Comment:
how much code is this function? Feels like starting a logger is pretty simple code, how about we just duplicate it here and then we don't need to deal with your urfave version issue?

Reply:
[8645e79](https://github.com/Layr-Labs/eigenda-proxy/pull/264/commits/8645e79550d48c8c99d67ea0460a4d1f729018e3)